src/SFML/Window/Win32/WindowImplWin32.cpp

```diff
@@ -497,8 +497,9 @@ void WindowImplWin32::switchToFullscreen(const VideoMode& mode)
     }
 
     // Make the window flags compatible with fullscreen mode
-    SetWindowLongW(m_handle, GWL_STYLE, WS_POPUP | WS_CLIPCHILDREN | WS_CLIPSIBLINGS);
-    SetWindowLongW(m_handle, GWL_EXSTYLE, WS_EX_APPWINDOW);
+    SetWindowLongPtrW(m_handle, GWL_STYLE, WS_POPUP | WS_CLIPCHILDREN | WS_CLIPSIBLINGS);
+    SetWindowLongPtrW(m_handle, GWL_EXSTYLE, WS_EX_APPWINDOW);
+    ShowWindow(m_handle, SW_SHOW);
```

Comment:
Why is there an additional `ShowWindow()` call?

Reply:
@eXpl0it3r,
Did you check my PR in your displaying window was normal without ShowWindow()?

Reply:
I didn't, but if the mentioned function is a 1:1 replacement, then you shouldn't require additional code, right?

Reply:
@eXpl0it3r,
yes you u gd right